src/query/api/v1/handler/prometheus/handleroptions/headers.go

```diff
@@ -60,6 +60,10 @@ const (
 	// in JSON format. See `handler.stringTagOptions` for definitions.`
 	RestrictByTagsJSONHeader = M3HeaderPrefix + "Restrict-By-Tags-JSON"
 
+	// MapTagsByJSONHeader provides the ability to mutate tags of timeseries in
+	// incoming write requests. See `MapTagsOptions` for structure.
+	MapTagsByJSONHeader = "M3-Map-Tags-By-JSON"
```

Comment:
Can you make this `M3HeaderPrefix + "Map-Tags-JSON"`? Can we remove the word "By" here? For the restrict it's the term "Restrict-By" then "Tags" then "JSON" this is "Map" then "Tags" then the "By" is before JSON (which is strange).

I think it should probably be "Map-By-Tags-JSON" or just "Map-Tags-JSON".